Tests/SwiftParserTest/Assertions.swift

```diff
@@ -12,7 +12,7 @@
 
 import XCTest
 @_spi(RawSyntax) import SwiftSyntax
-@_spi(Testing)@_spi(RawSyntax) import SwiftParser
+@_spi(Testing) @_spi(RawSyntax) import SwiftParser
```

Comment:
When running the formatter it add spacing here

Reply:
That's because you are running the formatter from ˋmainˋ but CI is running the formatter from ˋrelease/5.9ˋ.

Reply:
I see. Will revert